c4p2n/models.py

```diff
@@ -62,6 +74,26 @@ class CallForPaperFormResponse(BaseModel):
     answers: List[Union[TextAnswer, MultipleChoiceAnswer]]
 
 
+class CallForPaperAnswers(BaseModel):
+    name: str
+    job: str
+    photo_link: str
+    talk_date: List[str]
+    telegram: str = Field(default="")
+    contact: str = Field(default="")
+    phone: str
+    talk_title: str = Field(default="")
+    talk_description: str
+    questions: str = Field(default="")
+
+    def talk_dates(self) -> str:
```

Comment:
I'm not sure why talk_dates isn't validator for talk_date? Maybe property? Or rewrite `MultipleChoiceAnswer.extract` to `-> str`

Reply:
Started using property.